tests/TestCase/TestSuite/FixtureManagerTest.php

```diff
@@ -54,6 +56,37 @@ public function testFixturizeCore()
     }
 
     /**
+     * Test logging depends on fixture manager debug.
+     *
+     * @return void
+     */
+    public function testLogSchemaWithDebug()
+    {
+        $db = ConnectionManager::get('test');
+        $restore = $db->logQueries();
+        $db->logQueries(true);
+
+        $this->manager->setDebug(true);
+        $buffer = new ConsoleOutput();
+        Log::config('testQueryLogger', [
+            'className' => 'Console',
+            'stream' => $buffer
+        ]);
+
+        $test = $this->getMock('Cake\TestSuite\TestCase');
+        $test->fixtures = ['core.articles'];
+        $this->manager->fixturize($test);
+        // Need to load/shutdown twice to ensure fixture is created.
+        $this->manager->load($test);
+        $this->manager->shutdown();
+        $this->manager->load($test);
+        $this->manager->shutdown();
```

Comment:
The fixture could exist from an earlier test case that also included the fixture.